Stripe/STPPaymentOptionsInternalViewController.m

```diff
@@ -301,8 +300,18 @@ - (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEd
         // Perform deletion animation for single row
         [tableView deleteRowsAtIndexPaths:@[indexPath] withRowAnimation:UITableViewRowAnimationAutomatic];
 
+        BOOL tableViewIsEditing = tableView.isEditing;
+        if (![self isAnyPaymentOptionDetachable]) {
+            // we deleted the last available payment option, stop editing
+            [self _endTableViewEditing];
```

Comment:
The docs for this delegate method say we shouldn't do this: 

> You should not call setEditing:animated: within an implementation of this method. If for some reason you must, invoke it after a delay by using the performSelector:withObject:afterDelay: method.

Reply:
Ah, thanks for checking! I'll update